.github/config/openlane.txt

```diff
@@ -1,2 +1,2 @@
-OPENLANE_TAG=2.0.0
+OPENLANE_TAG=2.0.7
```

Comment:
I'd probably start from 2.0.5 instead since it would avoid a lot of unrelated "revert" in the openlane2.path.